src/main/java/edu/hawaii/its/api/wrapper/FindAttributesCommand.java

```diff
@@ -26,6 +26,11 @@ public FindAttributesCommand assignSearchScope(String scope) {
         return this;
     }
 
+    public FindAttributesCommand addAttribute(String attribute) {
+        this.gcFindAttributeDefNames.addAttributeDefNameName(attribute);
+        return this;
+    }
+
```

Comment:
Is this change being used? If not let's remove it.

Reply:
Ok, I'll remove this function. This was used when I was still exploring the functions of Gc.